base/applications/clipbrd/clipbrd.c

```diff
@@ -733,8 +748,16 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPWSTR lpCmdLi
     }
 
     /* If the user provided a path to a clipboard data file, try to open it */
-    if (lpCmdLine != NULL && *lpCmdLine)
-        LoadClipboardDataFromFile(lpCmdLine);
+    if (__argc >= 2)
+    {
+        if (MessageBoxRes(Globals.hMainWnd, Globals.hInstance,
+                          STRING_DELETE_MSG, STRING_DELETE_TITLE,
+                          MB_ICONWARNING | MB_YESNO) == IDYES)
+        {
+            if (!LoadClipboardDataFromFile(__wargv[1]))
```

Comment:
What does this try to do? (calling twice the LoadClipboard.... function)

Reply:
__wargv[1] is the (maybe-quoted) 1st argument. lpCmdLine is the whole command line. They are different.

If the command line was <clipbrd "abc def" ghi>, then __wargv[1] will be abc def, and lpCmdLine will be "abc def" ghi.

If the command line was <clipbrd abc def ghi>, then __wargv[1] will be abc, and lpCmdLine will be abc def ghi.